pytato/loopy.py

```diff
@@ -151,13 +152,12 @@ def __iter__(self) -> Iterator[str]:
         return iter(self._result_names)
 
 
-@attrs.frozen(eq=False, hash=True, cache_hash=True)
+@array_dataclass()
 class LoopyCallResult(NamedArray):
```

Comment:
Here is a workaround for the mypy error: 

```suggestion
@array_dataclass()
class LoopyCallResult(NamedArray):  # type: ignore[override] # https://github.com/python/mypy/issues/17623
```

Reply:
Thanks! I was in the process of falling down the same dumb rabbithole.